### ISSUE TITLE
Games That uses CRO

### ISSUE THREAD

**Issue by HyperGainZ** [wwylele/citra#4]
Inazuma Eleven 3 - Bomb Blast - [LOG](http://hastebin.com/ukamanicaq.xml)

Inazuma Eleven 3 - Team Ogre Attacks [LOG](http://hastebin.com/eliyuxoxuk.xml)

Inazuma Eleven 3 - Lightning Bolt [LOG](http://hastebin.com/eyuwodaziv.xml)

Super Smash Bros. - [LOG](https://ghostbin.com/paste/dbpxj)

**Comment by HyperGainZ** [comment]
Super Smash Bros - FIXED was missing nand Dump 
https://github.com/citra-emu/citra/wiki/Home-Folder

**Comment by wwylele** [comment]
I remember you said Mario & Luigi Dream Team can boot? Could you please play it for a while to see if there are more issues?
Also, do you have _Puzzle & Dragons Z_ and _Puzzle & Dragons Super Mario Bros. Edition_? (https://github.com/citra-emu/citra/issues/1592)

Please pay special attention to all Service.LDR error (warnings and infos are OK and can be ignored) when testing.

**Comment by HyperGainZ** [comment]
@wwylele il do the tests of those games you named above and give the logs even if the games play well

**Comment by HyperGainZ** [comment]
Mario & Luigi Dream Team - works - [LOG](http://hastebin.com/setexifebe.xml)  -- No Service.LDR errors

 Puzzle & Dragons Z - dont works - Log is to large to paste -- `[   4.051104] HW.Memory <Error> core/memory.cpp:Read:181: unmapped Read32 @ 0x00000738` 

 Super Mario Bros - works - [LOG](http://hastebin.com/mowaqumeme.xml)  -- No Service.LDR errors

**Comment by Leo626** [comment]
Megaman Legacy Collection boots in the nightly builds but spams this whenever a game is selected:
```
[  80.814398] Debug.Emulated <Critical> core\hle\svc.cpp:SVC::Break:406: Emulated program broke execution!
[  80.814531] Debug.Emulated <Critical> core\hle\svc.cpp:SVC::Break:414: Break reason: PANIC
```

With this, it won't boot and spams an error similar to Puzzles & Dragons Z:

`[  34.242089] HW.Memory <Error> core\memory.cpp:Memory::Read:181: unmapped Read32 @ 0x0003DFFC`

**Comment by Leo626** [comment]
Sorry forgot to post logs for [master](http://clipboard.space/clip/n7wYdKNJZRcRd755K9YX) and [cro++](http://clipboard.space/clip/NWqjr75y71CPqDZZYweO). According to the log from the master, there are unimplemented functions `LoadExeCRO_New` and `LoadCROSymbols`, while cro++ doesn't mention it.

**Comment by wwylele** [comment]
@Leo626  I finally found that Puzzles & Dragons Z is actually playable. It was freeze due to wrong region and language settings. Probably similar issue with Megaman Legacy Collection?

**Comment by Leo626** [comment]
The rom is USA region and I have the region set to USA in the configure widget. However, it still spams that error. I merged cro++ with set-config-block to ensure that the language is set to English but it's still the same. The current nightly boots it like normal.

**Comment by wwylele** [comment]
@HyperGainZ Could you please put the log of Mario & Luigi Dream Team? Just the log until the menu is OK. I'd like to see some detail information.

**Comment by HyperGainZ** [comment]
@wwylele here you go [LOG](http://hastebin.com/fimihuxugo.xml)

**Comment by wwylele** [comment]
I will close this once the CRO PR is merged into the main master branch. Any further issue can go to the main repo then
